### ISSUE TITLE
Multiple Files Upload with InputFile

### ISSUE THREAD

**Issue by denisprado** [Rocketseat/unform#135]
**Description of bug**

When I upload multiple files with FileInput component, the file-data atribute get only one base64 file. 

**To Reproduce**
I have used the component as follow:
```
<ModalForm onSubmit={handleSubmitFiles} encType="multipart/form-data">
      <FileInput
        type="file"
        name="files"
        id="files"
        multiple
        onStartProgress={handleProgress}
      />
      <Button type="submit"> Enviar </Button>
    </ModalForm>
```

**Expected behavior**

I expected several files to be uploaded and this would allow proper formatting with FormData.

**Screenshots**

![Captura de tela 2019-09-26 15 22 20](https://user-images.githubusercontent.com/10910579/65714960-898b1d80-e072-11e9-9924-f88f8fbc2e71.png)

**Comment by WitchMad** [comment]
Did you tested without "multiple" config?
That could show us if something else is wrong

**Comment by diego3g** [comment]
Hello @denisprado, how are you?

We've released Unform 2.0 🎉

With this, unform does not exposes a FileInput component anymore, neither any other pre built component, so the user must create it's own inputs before using unform itself, we explain this inside the [migration guide](https://unform.dev/migration-guide).

After you upgrade to Unform 2.0, you can create a MultipleFileInput using this code:

```jsx
import React, { useRef, useEffect }  from 'react';

import { useField } from '@unform/core';

interface Props {
  name: string;
}

type InputProps = JSX.IntrinsicElements['input'] & Props;

const MultipleFileInput: React.FC<InputProps> = ({ name, ...rest }) => {
  const inputRef = useRef<HTMLInputElement>(null);

  const { fieldName, registerField, defaultValue, error } = useField(name);

  useEffect(() => {
    registerField({
      name: fieldName,
      ref: inputRef.current,
      path: 'files',
      getValue(ref: HTMLInputElement) {
        return ref.files && Array.from(ref.files);
      },
      clearValue(ref: HTMLInputElement) {
        ref.value = '';
      },
    })
  }, [fieldName, registerField]);

  return (
    <>
      <input 
        multiple
        type="file" 
        ref={inputRef} 
        {...rest}
      />
    </>
  );
};

export default MultipleFileInput;
```

If you have any further questions please let me know.

I'll close the issue for now but feel free to reopen it or create new ones :)
